src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/List32FW.java

```diff
@@ -0,0 +1,146 @@
+    }
+
+    @Override
+    public int logicalLength()
+    {
+        return buffer().getInt(offset() + LOGICAL_LENGTH_OFFSET);
+    }
+
+    @Override
+    public int lengthSize()
+    {
+        return PHYSICAL_LENGTH_SIZE;
+    }
+
+    @Override
+    public DirectBuffer fields()
+    {
+        return fieldsRO;
+    }
+
+    @Override
+    public List32FW tryWrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        if (super.tryWrap(buffer, offset, maxLimit) == null)
+        {
+            return null;
+        }
+        int fieldsLength = physicalLength() - LOGICAL_LENGTH_SIZE;
+        fieldsRO.wrap(buffer, offset + PHYSICAL_LENGTH_SIZE + LOGICAL_LENGTH_SIZE, fieldsLength);
+        if (limit() > maxLimit)
+        {
+            return null;
+        }
+        return this;
+    }
+
+    @Override
+    public List32FW wrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        super.wrap(buffer, offset, maxLimit);
+        int fieldsLength = physicalLength() - LOGICAL_LENGTH_SIZE;
+        fieldsRO.wrap(buffer, offset + PHYSICAL_LENGTH_SIZE + LOGICAL_LENGTH_SIZE, fieldsLength);
+        checkLimit(limit(), maxLimit);
+        return this;
+    }
+
+    public static final class Builder extends ListFW.Builder<List32FW>
+    {
+        public Builder()
+        {
+            super(new List32FW());
+        }
+
+        @Override
+        public Builder set(
+            ListFW value)
+        {
+            int newLimit = offset() + PHYSICAL_LENGTH_SIZE;
+            checkLimit(newLimit, maxLimit());
+            buffer().putInt(offset(), value.physicalLength());
+            limit(newLimit);
+            fieldsLength(value.physicalLength() - LOGICAL_LENGTH_SIZE);
+
+            newLimit = limit() + LOGICAL_LENGTH_SIZE;
+            checkLimit(newLimit, maxLimit());
+            buffer().putInt(limit(), value.logicalLength());
+            limit(newLimit);
+            fieldsCount(value.logicalLength());
+
+            int fieldsSize = value.physicalLength() - LOGICAL_LENGTH_SIZE;
+            newLimit = limit() + fieldsSize;
+            checkLimit(newLimit, maxLimit());
+            buffer().putBytes(limit(), value.fields(), 0, fieldsSize);
+            limit(newLimit);
+            return this;
+        }
+
+        @Override
+        public Builder wrap(
+            MutableDirectBuffer buffer,
+            int offset,
+            int maxLimit)
+        {
+            super.wrap(buffer, offset, maxLimit);
+            int newLimit = offset + PHYSICAL_LENGTH_SIZE + LOGICAL_LENGTH_SIZE;
+            checkLimit(newLimit, maxLimit);
+            limit(newLimit);
+            return this;
+        }
+
+        @Override
+        public List32FW build()
+        {
+            buffer().putInt(offset() + PHYSICAL_LENGTH_OFFSET, fieldsLength() + LOGICAL_LENGTH_SIZE);
+            buffer().putInt(offset() + LOGICAL_LENGTH_OFFSET, fieldsCount());
```

Comment:
Since we don't have access to `sizeof()` method in `Builder` class, I replaced it with the followng:
```
buffer().putInt(offset() + PHYSICAL_LENGTH_OFFSET, limit() - offset() - LOGICAL_LENGTH_OFFSET);
```